manifests/m/Mozilla/Firefox/116.0/Mozilla.Firefox.locale.ta-IN.yaml

```diff
@@ -0,0 +1,37 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: "116.0"
+PackageLocale: ta-IN
+Publisher: Mozilla
+PublisherUrl: https://www.mozilla.org/en-US/
+PublisherSupportUrl: https://support.mozilla.org/en-US/
+PrivacyUrl: https://www.mozilla.org/en-US/privacy/websites
+Author: மொஸில்லா அறக்கட்டளை
+PackageName: Mozilla Firefox
+PackageUrl: https://www.mozilla.org/en-US/firefox/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0
+CopyrightUrl: https://www.mozilla.org/en-US/foundation/trademarks/policy
+ShortDescription: மொஸில்லா பயர்பாக்ஸ் என்பது இலவச மற்றும் திறந்த மூல மென்பொருளாகும், இது உலகம் முழுவதிலுமிருந்து ஆயிரக்கணக்கான சமூகத்தால் உருவாக்கப்பட்டது.
+Description: |-
+  பயர்பாக்ஸ் உலாவி, மொஸில்லா பயர்பாக்ஸ் அல்லது வெறுமனே பயர்பாக்ஸ் என்றும் அழைக்கப்படுகிறது, இது மொஸில்லா அறக்கட்டளை மற்றும் அதன் துணை நிறுவனமான மொஸில்லா கார்ப்பரேஷனால் உருவாக்கப்பட்ட ஒரு இலவச மற்றும் திறந்த மூல இணைய உலாவியாகும்.
+  பயர்பாக்ஸ் இணையப் பக்கங்களை வழங்குவதற்கு கெக்கோ தளவமைப்பு இயந்திரத்தைப் பயன்படுத்துகிறது, இது தற்போதைய மற்றும் எதிர்பார்க்கப்பட்ட இணையத் தரங்களைச் செயல்படுத்துகிறது.
+  2017 ஆம் ஆண்டில், இணையான மற்றும் மிகவும் உள்ளுணர்வு பயனர் இடைமுகத்தை மேம்படுத்துவதற்காக குவாண்டம் என்ற குறியீட்டு பெயரில் புதிய தொழில்நுட்பத்தை பயர்பாக்ஸ் இணைக்கத் தொடங்கியது.
+  Firefox Windows 7 அல்லது புதிய, macOS மற்றும் Linux க்கு அதிகாரப்பூர்வமாக கிடைக்கிறது.
+  அதன் அதிகாரப்பூர்வமற்ற போர்ட்கள் FreeBSD, OpenBSD, NetBSD, illumos மற்றும் Solaris Unix உள்ளிட்ட பல்வேறு Unix மற்றும் Unix போன்ற இயங்குதளங்களுக்கு கிடைக்கின்றன.
+Tags:
+- mozilla
+- browser
+- web
+- web-browser
+- foss
+- உலாவி
+- இணைய-உலாவி
+- குறுக்கு-தளம்
+- கெக்கோ
+- குவாண்டம்
+- சிலந்தி-குரங்கு
+ReleaseNotesUrl: https://www.mozilla.org/en-US/firefox/114.0.1/releasenotes/
```

Comment:
This looks like it needs updating

Reply:
I just removed it, wrong language anyway.